src/site/content/en/blog/user-agent-client-hints/index.md

```diff
@@ -190,7 +190,8 @@ are used for representing lists and booleans.
 | ------------------------------------------- | ------------------------------------------------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `Sec-CH-UA`                                 | `"Chromium";v="84",`<br>`"Google Chrome";v="84"` | List of browser brands and their significant version.                                                                                                                        |
 | `Sec-CH-UA-Mobile`                          | `?1`                                             | Boolean indicating if the browser is on a mobile device (`?1` for true) or not (`?0` for false).                                                                             |
-| `Sec-CH-UA-Full-Version`                    | `"84.0.4143.2"`                                  | The complete version for the browser.                                                                                                                                        |
+| `Sec-CH-UA-Full-Version`                    | `"84.0.4143.2"`                                  | [**Deprecated**]The complete version for the browser.                                                                                                                                        |
+| `Sec-CH-UA-Full-Version-List`               | `"Chromium";v="84.0.4143.2",`<br>`"Google Chrome";v="84.0.4143.2"` | List of browser brands and their full version.                                                                                                           |
```

Comment:
@rowan-m @yoavweiss Is there a particular reason we're breaking out Chrome and Chromium? I didn't think they could ever be different. Can't we just go with '84'. The 4th number always goes with a single platform build. Consequently listing the full version number implies something that isn't true.

Reply:
> Is there a particular reason we're breaking out Chrome and Chromium?

Joe, IIUC, as there are multiple browsers vendors using a same engine. For this case, Chrome is the browser version, and Chromium is the engine.  For edge, the `Sec-CH-UA-Full-Version-List` can be like following: 

```
Sec-CH-UA-Full-Version-List: "Microsoft Edge"; v="92.0.902.73", "Chromium"; v="92.0.4515.131", "?Not:Your Browser"; v="3.1.2.0"
```

Reply:
It's generally only the small 'c' chrome that's different. If it's in Chromium, it's in Chrome, Edge, and Opera. (Samsung is different.). A four part number does not ever refer to Chrome generally. It refers to one platform. Have a look [at this chart](https://omahaproxy.appspot.com/) to see what I'm talking about.

Reply:
could you elaborate the meaning of  `only the small 'c' chrome that's different` .  BTW,  for the context we are resolving the issue: https://github.com/WICG/ua-client-hints/issues/196. thanks.

Reply:
The user interface.

Reply:
@jpmedley can you clarify what changes you are wanting? I don't understand your comments

> Is there a particular reason we're breaking out Chrome and Chromium? I didn't think they could ever be different.

This is true for Chrome (today, anyways). But Edge has a different versioning scheme from the Chromium version. That's the point of this feature.